draft-ietf-ipsecme-ikev2-multiple-ke-00.xml

```diff
@@ -863,29 +864,29 @@ SKEYSEED = prf(SK_d, SK(0) | Ni | Nr | SK(1) | ... SK(n))
 KEYMAT = prf+ (SK_d, SK(0) | Ni | Nr | SK(1) |  ... SK(n))
                 ]]></artwork></figure>
 
-                <t> In both cases SK_d is from existing IKE SA; SK(0), Ni, Nr are the shared key and nonces
+                <t> In both cases, SK_d is from the existing IKE SA; SK(0), Ni, Nr are the shared key and nonces
                 from the CREATE_CHILD_SA respectively; SK(1)...SK(n) are the shared keys from additional key exchanges.
                 </t>
 
             </section>
 
             <section title="Interaction with Childless IKE SA">
-            <t>It is also possible to establish a fully quantum-resistant IKE SAs from additional key exchanges
+            <t>It is also possible to establish a fully post-quantum secure IKE SAs from additional key exchanges
             without using IKE_INTERMEDIATE exchanges.  In this case, the IKE SA created from IKE_SA_INIT exchange
-            can be immediately rekeyed with CREATE_CHILD_SA using additional key exchanges and IKE_FOLLOWUP_KE
-            message to carry the key exchange payload.  If only classical key exchange method is used in
-            the IKE_SA_INIT message, the very first Child SA created in IKE_AUTH will not be quantum resistant.
-            Consequently, if the peers' local policy requires that all Child SAs should be fully-protected, then
+            can be immediately rekeyed with CREATE_CHILD_SA using additional key exchanges where IKE_FOLLOWUP_KE
+            messages are used to carry the key exchange payload.  If classical key exchange method only is used in
+            the IKE_SA_INIT message, the very first Child SA created in IKE_AUTH will offer no resistance against the
+            quantum threats.  Consequently, if the peers' local policy requires that all Child SAs should be fully-protected, then
             the peers can avoid creating the very first Child SA by adopting <xref target="RFC6023"/>.  In this
             case, the peers exchange CHILDLESS_IKEV2_SUPPORTED notification in the IKE_SA_INIT exchange and a
             fully-protected Child SA can be created with CREATE_CHILD_SA using additional key exchanges.
             </t>
 
             <t>Note that if the initial IKE SA is used to transfer sensitive information, then this information
-            will not be protected using the additional (e.g. quantum safe) key exchanges, so this scenario
-            may be inappropriate.  One such example is in G-IKEv2 protocol <xref target="I-D.ietf-ipsecme-g-ikev2"/>
-            where cryptographic materials are exchanged in IKE_SA_INIT messages between group member and the group
-            controller.</t>
+            will not be protected using the additional key exchanges, which may use post-quantum algorithms.  Consider
+            G-IKEv2 protocol <xref target="I-D.ietf-ipsecme-g-ikev2"/> where cryptographic materials are exchanged in IKE_SA_INIT
```

Comment:
This is incorrect (my fault). In G-IKEv2 cryptographic material is sent from the GCKS to GM in the GSA_AUTH exchange. The correct text is:

Consider G-IKEv2 protocol <xref target="I-D.ietf-ipsecme-g-ikev2"/> where cryptographic materials are sent from the group controller to the group member at the time the initial IKE SA is being created. In this arrangement, the peers will have to use post-quantum algorithm in Transform Type 4 in order to mitigate the risk of quantum attack.

Reply:
Thanks @smyslov. It is sorted now.